bitcoin/psbt.c

```diff
@@ -713,7 +705,9 @@ const u8 *psbt_get_bytes(const tal_t *ctx, const struct wally_psbt *psbt,
 		return NULL;
 	}
 
-	wally_psbt_get_length(psbt, 0, &len);
+	if (wally_psbt_get_length(psbt, 0, &len) != WALLY_OK) {
```

Comment:
How does this fail?  Is this assert-worthy?

Reply:
I suppose prior behavior was likely returning 0, then failing later in `wally_psbt_to_bytes` which results in an abort. I can also abort here?

Reply:
changed to assert